css/style.css

```diff
@@ -150,3 +150,344 @@ a {
   max-width: 100%;
   display: block;
 }
+
+.myrecentwork {
+  width: 327px;
+  height: 97px;
+  padding: 22px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  position: relative;
+  display: grid;
+  align-items: center;
```

Comment:
- [ ] Kindly make sure your page is not overlapping as it is. It is supposed to fit 100% of the screen size and not more. as you can see in the provided Figma template
- [ ] Also you may consider using the same `typo(font-size, font family)` and `color` as the ones used in the Figma template.
Kindly fix that and you will be good to go.
Your design :point_down: 

![image](https://user-images.githubusercontent.com/97100091/198543388-02903f71-6d54-4bc1-a36c-15b3dd5bb959.png)
![image](https://user-images.githubusercontent.com/97100091/198543470-141e6793-6847-4a01-a426-e3d1203f3d4e.png)
 
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198543963-5799f2b7-e499-4af6-8f9f-595f1eb18ce4.png)

Reply:
Hello @christianbiring1 
## Actually I do not get you. kindly please would you explain it more. as I have attached in below my design looks fin and overlapped.  
![image](https://user-images.githubusercontent.com/90598109/198584000-1107c02a-8ee5-44b1-bb48-5144e3ad77fd.png)